src/main/java/cuchaz/enigma/translation/mapping/serde/RawEntryMapping.java

```diff
@@ -0,0 +1,30 @@
+package cuchaz.enigma.translation.mapping.serde;
+
+import cuchaz.enigma.translation.mapping.AccessModifier;
+import cuchaz.enigma.translation.mapping.EntryMapping;
+
+import java.util.ArrayList;
+import java.util.List;
+
+final class RawEntryMapping {
```

Comment:
Could take advantage of this to make it an `EntryMappingBuilder` where the other params are set via builder methods too

Reply:
The other two (access level and target name) are finalized on first parse while javadoc is mutable.